{"name": "testcase2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3\n  +2\n  -3", "trace": "@pytest.mark.run(order=2)\n    @allure.title(\"testcase2\")\n    @allure.description(\"I am first test_case2\")\n    def test_case2():\n        print('I am case2')\n        LoggerForApitest.writeInfoLog(\"I AM CASE2\")\n>       assert 2 == 3\nE       assert 2 == 3\nE         +2\nE         -3\n\npytest_case\\test_fixtures1.py:45: AssertionError"}, "description": "I am first test_case2", "attachments": [{"name": "log", "source": "5be9fd26-fdc9-4b89-8c0c-2a4cfa3ed3ec-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "052f5710-9c41-446a-91b5-76f317d3dd6e-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "338ca77d-1fef-41e4-98ee-ddb5631f13c4-attachment.txt", "type": "text/plain"}], "start": 1647935473769, "stop": 1647935473769, "uuid": "91547ac0-cef7-4cfe-8b16-ed6346db1064", "historyId": "92f9d3b642298a60c88279ecf235fa2d", "testCaseId": "725ce7cbcf31b4144284ab868efa6e15", "fullName": "pytest_case.test_fixtures1#test_case2", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "pytest_case"}, {"name": "suite", "value": "test_fixtures1"}, {"name": "host", "value": "no-delecs-g42-test-wx405299"}, {"name": "thread", "value": "14960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_case.test_fixtures1"}]}